packages/cli/generators/controller/templates/new.controller.ts

```diff
@@ -0,0 +1,8 @@
+// Uncomment these imports to begin using these cool features!
+/*
+import {inject} from @loopback/context;
```

Comment:
I was under the impression that the task was for creating an empty controller without any assumptions on the kind of server the application is going to be using. + on the single line comment

Reply:
If you call the command twice and if yeoman detects that the possibility of a file name conflict, it gives you a prompt on the options which include skip, overwrite, abort, etc. Originally, I wrote something that disables the prompt by hacking in something because yeoman ~sucks~ doesn't export the feature in their package, but I just decided to go with the built-in prompt with Raymond's suggestion.